crates/package_json/src/lib.rs

```diff
@@ -87,7 +81,22 @@ impl PackageJson {
             "author": "",
             "license": "ISC"
         });
-        let contents = serde_json::to_string_pretty(&package_json)?;
+        Ok(package_json)
+    }
+
+    fn to_string_prettify(package_json: &Value) -> Result<String, PackageJsonError> {
```

Comment:
Why don't you make this an instance method instead of a static one? 

```suggestion
    fn to_string_prettify(&self) -> Result<String, PackageJsonError> {
```

Reply:
We will invoke this in some static method, such as `write_to_file`, so might a static one is better?